docs/dictionary/command/export-snapshot.lcdoc

```diff
@@ -2,9 +2,9 @@ Name: export snapshot
 
 Type: command
 
-Syntax: export snapshot [from rect[angle] <rectangle>] [of <object(glossary)>] [(with | without) effects] ... | [from <object>] [{with|and} metadata <metadata>] to {file <filePath> | <container>} [as <format>] [with mask <maskFile>]
+Syntax: export snapshot [from rect[angle] <rectangle>] [of <object(glossary)>] [(with | without) effects] <suffix> | [from <object>] [{with|and} metadata <metadata>] to {file <filePath> | <container>} [as <format>] [with mask <maskFile>]
 
-Syntax: export snapshot [from rect[angle] <rectangle>] of <object(glossary)> [(with | without) effects] ... | [from <object>] [at size width, height] [{with|and} metadata <metadata>] to {file <filePath> | <container>} [as <format>] [with mask <maskFile>]
+Syntax: export snapshot [from rect[angle] <rectangle>] of <object(glossary)> [(with | without) effects] <suffix> | [from <object>] [at size width, height] [{with|and} metadata <metadata>] to {file <filePath> | <container>} [as <format>] [with mask <maskFile>]
```

Comment:
curly brackets denote an option list, square brackets denote 'optional'. So for example if the syntax is 
```
before [{a|b}] after
```
then 
- `before a after`
- `before b after`
- `before after`

are all valid. If the syntax was 
```
before {a|b} after
```
then `before after` would not be valid